src/edge/models/op.py

```diff
@@ -8,12 +8,13 @@ class Meta:
 
     # numerical values are stored in database, do not change unless you have a
     # plan for migrating data.
+
     RECOMBINATION = (1, "Homologous Recombination")
     CRISPR_DSB = (2, "CRISPR-Cas9 WT (Double Stranded Break)")
-    PCR_SEQ_VERIFICATION = (3, "PCR Product Sequence Verification")
+    SSR = (4, "Site Specific Recombinase")
```

Comment:
For my own curiousity, why did PCR_SEQ_VERIFICATION get removed?

Reply:
i don't think we use that anywhere